### ISSUE TITLE
Site Migration: Migration plugin instructions

### ISSUE THREAD

**Issue by markbiek** [Automattic/wp-calypso#87393]
**Note: This should be behind the `onboarding/new-migration-flow` flag.**

More context: paYKcK-41n-p2

Once plugin installation and Atomic migration is complete (see #87392), we'll want to take them to an instruction page which shows the user:

1. How to install the migration plugin on the source site.
2. What to do once the plugin is installed.
3. The migration key so it's easy to copy.

## Page Copy

We're going to need to write a decent amount of copy for this page. I've opened a Google Doc (search for 'Site Migration Instructions Copy') where we can write/discuss/edit the copy.

The goal for i1 is to just have step-by-step text instructions and maybe some links to the source and destination sites. We'll try and make things like the migration key, etc more automated as we have time.

**Comment by andres-blanco** [comment]
@sixhours can this issue be closed? or are we keeping this issue opened because we still need the copy?

**Comment by sixhours** [comment]
> are we keeping this issue opened because we still need the copy?

@andres-blanco Yeah, that was my thought. :)

**Comment by andres-blanco** [comment]
Adding to the initial list, we will also need to instruct the user to create credentials and provide a way to copy them. The migration plugin requires both: migration key & user/password

**Comment by sixhours** [comment]
Unassigning myself and marking as on hold until we have copy/instructions.

**Comment by markbiek** [comment]
I'm unblocking this since we need to write the first draft of the instructions. I've updated the issue to include the initial goals for the copy.

**Comment by andres-blanco** [comment]
I've been looking into the plugin and found that it's pretty easy to get the migration key. It can be called via cli
```
wp migrateguru get_connection_key
```

And probably can be easily called in PHP code, [here's an example](https://plugins.trac.wordpress.org/browser/migrate-guru/trunk/wp_cli.php?rev=3041664#L21) and [this is](https://plugins.trac.wordpress.org/browser/migrate-guru/trunk/migrateguru.php?rev=3041664#L50) how to instatiate the bvinfo class.

**Comment by sixhours** [comment]
Just as a heads-up, the Google Doc mentioned in the OP has a draft of the documentation as well as an abridged version of the instructions for this screen.
